runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -919,24 +919,32 @@ gcParseSovereignArguments(J9JavaVM *vm)
 		goto _error;
 	}
 
-	result =  option_set_to_opt_percent(vm, "-Xmaxt", &index, EXACT_MEMORY_MATCH, &extensions->heapExpansionGCTimeThreshold);
-	if (OPTION_OK != result) {
-		if (OPTION_MALFORMED == result) {
-			j9nls_printf(PORTLIB, J9NLS_ERROR, J9NLS_GC_OPTIONS_MUST_BE_NUMBER, "-Xmaxt");
-		} else {
-			j9nls_printf(PORTLIB, J9NLS_ERROR, J9NLS_GC_OPTIONS_PERCENT_OUT_OF_RANGE, "-Xmaxt", 0.0, 1.0);
+	
+	if (-1 != FIND_ARG_IN_VMARGS(EXACT_MEMORY_MATCH, "-Xmaxt", NULL)) {
+		extensions->heapExpansionGCRatioThreshold._wasSpecified = true;
```

Comment:
I think conceptually it is make sense to set `_wasSpecified` for correctly formed (parsed) options only. Otherwise it creates "grey" area for options specified but correspondent value is not set because can not be parsed (and we need to rely on error handling to avoid this situation). I believe JVM has mode to ignore incorrect/unknown options so it can lead to unpredictable results (like specified but not set)

Reply:
I see. So what you're suggesting then is to move the line 
`extensions->heapExpansionGCRatioThreshold._wasSpecified = true;`
down to a new "else" case below, which would be equivalent to 
`if (OPTION_OK == result)`, is that what you are hoping for?

Reply:
Yes, this is correct. And look like there is no need to extra `FIND_ARG_IN_VMARGS()` call,  `option_set_to_opt_percent()` does it internally